Makefile

```diff
@@ -44,6 +44,11 @@ all: controller kubeseal
 
 generate: $(GO_FILES)
 	$(GO) mod vendor && $(GO) generate $(GO_PACKAGES)
+	@# code-generator insists in generating the file under diretory:
+	@# github.com/bitnami-labs/sealeds-secrets
+	@# instead of just updating ./pkg for that reason we generate at gentmp
+	@# and then move to ./pkg
+	cp -r gentmp/github.com/bitnami-labs/sealed-secrets/pkg . && rm gentmp/ -rf
```

Comment:
This looks tricky. I've been playing around and it seems to work, but I haven't found a cleaner way of doing this.

Maybe we should expend some time on this in the future.

Reply:
Let me add a TODO there...